<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QWidget" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База с заявками и кандидатами v0.01</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить заявку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Экспортировать заявки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export_candidates">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Экспортировать кандидата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Номер заявки</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <double>0</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_candidate_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Имя кандидата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_candidate_role">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Роль кандидата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_candidate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить кандидата</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
